statsmodels/sandbox/regression/gmm.py

```diff
@@ -1621,12 +1621,6 @@ class IVGMMResults(GMMResults):
     def fittedvalues(self):
         return self.model.predict(self.params)
```

Comment:
I _think_ this can also go, but it isn't 100% obvious that this is equivalent to `self.predict()`.